src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -432,6 +432,7 @@ protected function describeContainerEnvVars(array $envs, array $options = [])
                         ['<info>Default value</>', $env['default_available'] ? $dump($env['default_value']) : 'n/a'],
                         ['<info>Real value</>', $env['runtime_available'] ? $dump($env['runtime_value']) : 'n/a'],
                         ['<info>Processed value</>', $env['default_available'] || $env['runtime_available'] ? $dump($env['processed_value']) : 'n/a'],
+                        ['<info>Usage count</>', $env['usage_count'] ? $dump($env['usage_count']) : 'n/a'],
```

Comment:
For a count, `0` is correct.
```suggestion
                        ['<info>Usage count</>', $dump($env['usage_count'])],
```